dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/internal/localization/i18n/TraceLocalizationResource.nlsprops

```diff
@@ -0,0 +1,1024 @@
+###############################################################################
+# Copyright (c) 2014 IBM Corporation and others.
```

Comment:
Add 2023

Reply:
A nlsprops verification tool in the build does not currently allow the copyright to be updated.